test/requests/workspace_symbol_test.rb

```diff
@@ -105,4 +105,16 @@ class Foo
     assert_equal(1, result.length)
     assert_equal("Foo", T.must(result.first).name)
   end
+
+  def test_returns_method_symbols
+    @index.index_single(RubyIndexer::IndexablePath.new(nil, "/fake.rb"), <<~RUBY)
+      class Foo
+        def bar; end
```

Comment:
Let's also add an example for `def initialize`?

Reply:
Done!